### PR TITLE
Formula - aws delete sqs-item

### PR DESCRIPTION
**Pull request by antoniofilhozup**
# Pull Request

## What I did
A new formula to delete an item from Amazon SQS.

## How I did it

Using AWS CLI

## How to verify it
Run `rit aws delete sqs-item`

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

## Description for the changelog

Add `rit aws delete sqs-item` formula

### CODE REVIEW

File: aws/delete/sqs-item/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Ritchie Formula
```
Comment (GuillaumeFalourd) [inline aws/delete/sqs-item/README.md:1]: This formula README file isn't following the new README pattern ([example](https://github.com/ZupIT/ritchie-formulas/blob/master/aws/add/terraform-eks/README.md)). Could you update it please? 👍🏼

File: aws/delete/sqs-item/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN curl -fsSL https://commons-repo.ritchiecli.io/install.sh | bash
+
+RUN mkdir /rit
+COPY . /rit
+RUN sed -i 's/\r//g' /rit/set_umask.sh
+RUN sed -i 's/\r//g' /rit/run.sh
+RUN chmod +x /rit/set_umask.sh
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && unzip awscliv2.zip && ./aws/install
```
Comment (GuillaumeFalourd) [inline aws/delete/sqs-item/Dockerfile:13]: Question: I see you download the `.zip` file and unzip it to install aws.

I've seen other option online such as the following that looks easier and less verbose. Would it be an option to use something like this on the Dockerfile (with the rest of RIT configurations) ?

```FROM python:3-alpine

RUN pip3 --no-cache-dir install --upgrade awscli
```

**Review by GuillaumeFalourd — CHANGES_REQUESTED** [review]

File: aws/delete/sqs-item/README.md (hunk 1 above)
Comment (antoniofilhozup) [reply aws/delete/sqs-item/README.md]: done

**Review by GuillaumeFalourd — APPROVED** [review]
👍🏼

**Pull request merged**